lazarus dependencies pulls in libQt5Pas but not libQt5Pas-devel

<!-- Don't request update of package. We have a script for that. https://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt . However, a quality pull request may help. -->
### System

* xuname:  
  Void 5.5.15_1 x86_64 GenuineIntel uptodate rrrrmdFFFFF
* package:  
  lazarus-2.0.2_1

### Expected behavior
 Lazarus requires the -devel version of the libQt5Pas package in order to compile anything using a form...

### Actual behavior
when trying to "run" a project you see
linker: /usr/bin/ld: cannot find -lQt5Pas

### Steps to reproduce the behavior
install lazarus, "run" a project

--- a/srcpkgs/lazarus/template
+++ b/srcpkgs/lazarus/template
@@ -1,12 +1,12 @@
 # Template file for 'lazarus'
 pkgname=lazarus
 version=2.0.2
-revision=1
+revision=2
 archs="x86_64 i686"
 wrksrc=lazarus
 hostmakedepends="fpc rsync"
 makedepends="fpc-src qt5pas-devel"
-depends="fpc-src fpc gdb perl"
+depends="fpc-src fpc gdb perl qt5pas-devel"
 short_desc="Delphi-like IDE for FreePascal"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, MPL-2.0, LGPL-2.0-or-later"
@@ -61,4 +61,3 @@ do_install() {
 	vinstall install/lazarus.desktop 644 usr/share/applications
 	vinstall images/ide_icon48x48.png 644 usr/share/pixmaps lazarus.png
 }
-

